srcpkgs/btrfs-progs/template

```diff
@@ -55,6 +56,7 @@ libbtrfs-devel_package() {
 
 libbtrfsutil_package() {
 	short_desc+=" - btrfsutil library"
+	shlib_provides="libbtrfsutil.so.1"
```

Comment:
why are you declaring them manually? the hook should do this automagically...

Reply:
I thought this should be added. Sorry, I'm a newbie.